### DISCUSSION TITLE
제페토 월드에서 시점 변경

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#210] [5. Q&A]
안녕하세요.
빌드잇으로 생성한 월드에는 설정에 시점 변경이 가능한데,
유니티로 생성한 월드에서 해당 기능을 구현해서 넣어도 될까요?
UI 토글을 통해 자신의 아바타를 없애고 카메라 위치를 변경하려고 합니다.
확인 부탁드립니다.

![image](https://user-images.githubusercontent.com/5335151/149318524-fb68e1eb-968f-447b-b53d-8ebd00c6f96b.png)

**Comment by moondory77-naverz** [comment]
안녕하세요. 
현재 해당 기능은 캐릭터 컨트롤러 모듈에서 직접적으로 제공되고 있지 않습니다.
필요 시, 직접 구현이 필요합니다.

지속적으로 모듈 레벨에서 여러가지 편의기능을 도입하고 있는 상태이며
의견주신 기능 또한 검토해보도록 하겠습니다. 

감사합니다.

**Reply by strangestraw** [reply]
네 직접 구현해서 사용하면 되는 것으로 이해했습니다. 감사합니다.
